cosmic_ray/work_db.py

```diff
@@ -169,9 +171,9 @@ def use_db(path, mode=WorkDB.Mode.create):
       FileNotFoundError: If `mode` is `Mode.open` and `path` does not
         exist.
     """
-    db = WorkDB(path, mode)
+    database = WorkDB(path, mode)
     try:
-        yield db
+        yield database
+        database.close()
```

Comment:
It looks like this call to `close` should probably be in a `finally` block. I'm not sure why it was in the `except` block to begin with...